tests/integration/buildcmd/test_build_terraform_applications.py

```diff
@@ -336,20 +242,25 @@ def test_build_and_invoke_lambda_functions(self, function_identifier, expected_o
             if should_override_code:
                 command_list_parameters[
                     "container_env_var"
-                ] = "TF_VAR_hello_function_src_code=./artifacts/HelloWorldFunction2"
+                ] = "TF_VAR_HELLO_FUNCTION_SRC_CODE=./artifacts/HelloWorldFunction2"
```

Comment:
use a capital TF variables so I can run the same test cases in  windows. In windows all Environment variables are in Capital Letters